tests/test_movingtpf.py

```diff
@@ -319,3 +325,50 @@ def test_to_lightcurve():
     for t in range(len(target.time)):
         if target.pixel_quality[t][target.aperture_mask[t]].any() != 0:
             assert target.lc["aperture"]["quality"][t] > 0
+
+    # Check flux fraction has expected values
+    assert (target.lc["aperture"]["flux_fraction"] < 1).all()
+    assert np.isfinite(target.lc["aperture"]["flux_fraction"]).all()
+
+
+def test_make_lc():
+    """
+    Check that make_lc() correctly saves the LCF, that the file has the expected attributes
+    and that it can be opened with lightkurve.
+    """
+
+    # Make TPF for asteroid 1998 YT6
+    target = MovingTPF.from_name("1998 YT6", sector=6)
+    target.make_tpf()
+    target.make_lc(save=True, outdir="tests")
+
+    # Check the file exists
+    assert os.path.exists("tests/tess-1998YT6-s0006-1-1-shape11x11_lc.fits")
+
+    # Open the file with astropy and check some attributes
+    with fits.open("tests/tess-1998YT6-s0006-1-1-shape11x11_lc.fits") as hdul:
+        assert "BAD_BITS" in hdul[0].header.keys()
+        assert hdul[0].header["PROCVER"].strip() == __version__
+        assert hdul[0].header["AP_TYPE"].strip() == "prf"
+        assert hdul[0].header["BG_CORR"].strip() == "rolling"
+        assert hdul[0].header["VMAG"] > 0
+        assert "SPOCDATE" in hdul[0].header.keys()
+        assert "TIME" in hdul[1].columns.names
+        assert "FLUX" in hdul[1].columns.names
+        assert np.isnan(hdul[1].data["PSF_FLUX"]).all()
+        assert "MOM_CENTR1" in hdul[1].columns.names
+        assert "RA" in hdul[1].columns.names
+        assert "EPHEM1" in hdul[1].columns.names
```

Comment:
I didn't want to test every column, so I only chose a sample. Let me know if you think I need to check for more.

Reply:
If checking for one of these columns means that the "2" will be there too, because both are produced in the same line of code, so you are testing the same line, then that's fine.

Reply:
These parameters are all in pairs, so I'll keep it as is.